model.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+import calendar
+import pickle
+from sklearn.externals import joblib
+from sklearn.svm import SVR
+#from tensorflow.contrib import skflow
+
+from sklearn.model_selection import GridSearchCV, RandomizedSearchCV
+from sklearn.cross_validation import train_test_split
+from sklearn.metrics import r2_score, mean_squared_error
+from math import sqrt
+from scipy.stats import randint as sp_randint
+from scipy.stats import uniform as sp_uniform
+
+def build_train():
+
+	bike_data = pd.read_csv("datasets/bike-sharing.csv")
+	features = bike_data.columns[:-3]
+	target = bike_data.columns[-1]
+	X = bike_data[features.drop(['dteday'],['instant'])] # feature values 
+	y = bike_data[target]
+	X_train, X_test, y_train, y_test = train_test_split(X, y)
+	svr = SVR()
+	svr.fit(X_train, y_train)
+	svr_pred = svr.predict(X_test)
+
+	score_svr = r2_score(y_test, svr_pred)
+	rmse_svr = sqrt(mean_squared_error(y_test, svr_pred))
+
+	print(svr_pred)
+	return(svr)
+
+if __name__ == '__main__':
```

Comment:
Can you integrate this file as a module rather than a separate file that has to be trained! Maybe when you are checking if the model is trained or not, you could probably train the model by calling the function in the main web app itself!